open-company/open-company-email
# Fin / Growth change description in emails

https://trello.com/c/GJ1rkbkb

This PR adds change descriptions to finance and growth output in stakeholder update emails.

Uses: https://github.com/open-company/open-company-lib/pull/2

If there was data in the period contiguous to the most recent period, then the change since the last period is described in currency (cash) and in percentage (revenue, expenses/costs, growth metrics).

To test:

Review the code.

Run all the sample generation of updates in the `content.clj` comment section at the bottom. Compare the output to what should be expected from the data. Notice especially snapshots such as `sparse.json` and `bago-no-symbol.json` and `blanks-test.json` which have trickier data.

Temporarily adjust the data in the JSON files your testing to challenge the output (no change at all, huge change, very tiny change, etc.).

Merge.

Deploy to beta.

Kneel during the playing of the national anthem as a form of social protest.
